corehq/apps/app_manager/templates/app_manager/partials/mobile_report_configs.html

```diff
@@ -60,7 +60,22 @@
                             <input type="text" data-bind="value: display">
                         </td>
                         <td>
-                            <input type="text" data-bind="value: description">
+                            <p>
```

Comment:
small tweak: changing this from a `p` to a `label` with class `checkbox` will a) align the box and the text and b) make the text clickable: http://getbootstrap.com/2.3.2/base-css.html#forms